rfd/0159-webasset-cache.md

```diff
@@ -135,30 +103,35 @@ if strings.HasPrefix(r.URL.Path, "/web/app") {
 		http.StripPrefix("/web", fs).ServeHTTP(w, r)
         return
 	}
-    res, err := h.auth.proxyClient.GetWebasset(cfg.Context, &webassetcache.GetWebassetRequest{
-		Name: r.URL.Path[len("/web/app/"):],
-	})
+	writer := &MemoryWriterAt{}
+	cfg.WebassetHandler.Download(cfg.Context, session.ID(fmt.Sprintf("%s/%s", teleport.Version, fileName)), writer)
```

Comment:
What does a `session.ID` have to do with webassets?

Reply:
Vestigial from some exiting code I took from the session uploader. Will remove from the example (it's not in the current PoC)